inputs/prod_customize.json

```diff
@@ -0,0 +1,7 @@
+{
+    "disable_plugins": [
+    ],
+
+    "enable_plugins": [
+    ]
+}
```

Comment:
Just don't want it overwriting a prod_customize.json file that's been carefully altered.